### ISSUE TITLE
can't compile

### ISSUE THREAD

**Issue by isothiazol** [gwihlidal/vk-mem-rs#65]
i tried to use this crate, but got this error when trying to compile a project:

`error occurred: Command "c++" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-gdwarf-4" "-fno-omit-frame-pointer" "-m64" "-I" "vendor/src" "-I" "wrapper" "-I" "wrapper/vulkan" "-Wall" "-Wextra" "-std=c++11" "-Wno-missing-field-initializers" "-Wno-unused-variable" "-Wno-unused-parameter" "-Wno-unused-private-field" "-Wno-reorder" "-DVMA_STATIC_VULKAN_FUNCTIONS=0" "-o" "/home/arch/code/rust/vkl/target/debug/build/vk-mem-3c04eecf1e16ee55/out/wrapper/vma_lib.o" "-c" "wrapper/vma_lib.cpp" with args "c++" did not execute successfully (status code exit status: 1).`

i tried overriding the compiler by changing the CXX env variable,
but i got the same error.
what should I do?

**Comment by colinmarc** [comment]
I have the following (presumably the same) compilation error with GCC 13:

```
  cargo:warning=vendor/VulkanMemoryAllocator/include/vk_mem_alloc.h:2817:9: error: ‘snprintf’ was not declared in this scope
  cargo:warning= 2817 |         snprintf(outStr, strLen, "%u", static_cast<unsigned int>(num));
  cargo:warning=      |         ^~~~~~~~
  cargo:warning=vendor/VulkanMemoryAllocator/include/vk_mem_alloc.h:2651:1: note: ‘snprintf’ is defined in header ‘<cstdio>’; did you forget to ‘#include <cstdio>’?
  ```

This was fixed upstream: https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator/commit/383f06b9fd9d36fb54cb3d4ac3363a670104bc62

So I think all that's needed is a version bump.

**Comment by Neo-Zhixing** [comment]
Should be resolved in #66
